src/configure/configure.ts

```diff
@@ -493,7 +493,7 @@ class Orchestrator {
                     this.inputs.sourceRepository.localPath,
                     this.inputs.sourceRepository.repositoryProvider,
                     repoAnalysisResult,
-                    this.inputs.pipelineConfiguration.params[constants.TargetResource])
+                    this.inputs.githubPATToken)
```

Comment:
was this a bug?

Reply:
This previous parameter was not being used in the function, so removed that, and passed GitHubPat as now, that is required.